src/Avalonia.Themes.Default/Accents/BaseDarkSimple.xaml

```diff
@@ -0,0 +1,71 @@
+<Style xmlns="https://github.com/avaloniaui"
+       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+       xmlns:sys="clr-namespace:System;assembly=netstandard">
+    <Style.Resources>
```

Comment:
your other changes have an indent of 2 spaces this whole file has 4

Reply:
file removed.